src/PatchOrchestrationApplication/NodeAgentNTService/src/Manager/WindowsUpdateManager.cs

```diff
@@ -29,6 +29,7 @@ class WindowsUpdateManager
         private UpdateSession _uSession;        
         private WUCollectionWrapper _wuCollectionWrapper;
         private Task<bool> _task;
+        private const string WUOperationStatusUpdate = "WUOperationStatusUpdate";
```

Comment:
>WUOperationStatusUpdate [](start = 56, length = 23)

How is this property different from WindowsUpdateStatus?

Reply:
WindowsUpdateStatus was previously being used by us to post events on NodeAgentService.